strictdoc/export/html/form_objects/document_grammar_form_object.py

```diff
@@ -50,11 +50,13 @@ def __init__(
         field_name: str,
         field_required: bool,
         reserved: bool,
+        field_human_title: Optional[str] = None,
     ):
         self.field_mid: str = field_mid
         self.field_name: str = field_name
         self.field_required: bool = field_required
         self.reserved: bool = reserved
+        self.field_human_title: Optional[str] = field_human_title
```

Comment:
Please make the order of both arguments and fields as follows:

```
self.field_name: str = field_name
self.field_human_title: Optional[str] = field_human_title
self.field_required: bool
self.reserved: bool = reserved
```

This way you would maintain a better rhythm.

Reply:
I thought the same about this, but changing the order runs into a pylint issue:

`error: Failed to parse strictdoc/export/html/form_objects/document_grammar_form_object.py:44:9: non-default argument follows default argument`

Reply:
Are you saying that you get this pylint issue if you have it like this:

```py
 def __init__(
        self,
        field_mid: str,
        field_name: str,
        field_human_title: Optional[str], 
        field_required: bool,
        reserved: bool,
    ):
```

?

Reply:
Yes! Actually the interpreter gives you an issue as well. 

I just found [this solution](https://dev.to/rajeshj3/solved-non-default-argument-follows-default-argument-400d) which I have never seen before introduced in python 3.4:


```python
def greet(*, name='Nicolas', place):
    return f"Welcome {name}, to {place}"
``` 

with an "*" at the beginning.